hal/halx86/acpi.cmake

```diff
@@ -1,13 +1,14 @@
 
 list(APPEND HAL_ACPI_SOURCE
+    include/hal.h
```

Comment:
Don't we usually have the PCH last in the list?

Reply:
It shouldn't matter, but for consistency with the rest we could put it in the end yeah. It just made more sense to start the unified source file with the.. header, you know :)